### PR TITLE
New tool to profile ops in model

### PR DESCRIPTION
**Pull request by Ac2zoom**

### CODE REVIEW

File: tools/profile_model_ops.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# A tool to profile the ops in ONNX models
+# Run by providing path to an ONNX model as the only CLI arg
+# A set of all unique ops present in the model will be printed
+
+import onnx
+import sys
+
```
Comment (houseroad) [inline tools/profile_model_ops.py]: Probably have main function and other functions?

We can do more profile on the model, not only op_types, we can keep it extensible

File: tools/profile_model_ops.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+if len(sys.argv) == 2:
```
Comment (bddppq) [inline tools/profile_model_ops.py]: use argparse :-)

File: tools/profile_model_ops.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import sys
+import argparse
+
+
+def uniqueOps(model):
+    print(set([n.op_type for n in model.graph.node]))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Calculate stats on ONNX models')
```
Comment (houseroad) [inline tools/profile_model_ops.py]: add an arg, like --unique-op?

**Comment by bddppq** [comment]
```
+python setup.py typecheck
running typecheck
tools/profile_model_ops.py:10: error: Function is missing a type annotation
```

Need to add type annotations

**Comment by xadupre** [comment]
Do you have other features in mind for this tool?

File: tools/profile_model_ops.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# Run by providing path to an ONNX model as the only CLI arg
+# A set of all unique ops present in the model will be printed
+
+import onnx
+import sys
+import argparse
+
+
+def uniqueOps(model):  # type: (onnx.onnx_pb2.ModelProto) -> None
+    print(set([n.op_type for n in model.graph.node]))
```
Comment (xadupre) [inline tools/profile_model_ops.py:11]: Recursivity? (control flow) What about functions?

**Pull request closed without merging**